ingestion_tools/scripts/importers/base_importer.py

```diff
@@ -214,6 +214,11 @@ def _finder_execution(
                 for item in source_finder_factory.find(config, metadata, **parents):
                     yield item
 
+    def __repr__(self) -> str:
```

Comment:
Added to provide more meaningful names for the objects when printed in the allure reports.